manifests/c/calibre/calibre/6.2.1/calibre.calibre.installer.yaml

```
@@ -0,0 +1,39 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: calibre.calibre
+PackageVersion: 6.2.1
+Platform:
+  - Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+  - azw
+  - azw3
+  - docx
+  - epub
+  - fb2
+  - fbz
+  - html
+  - lit
+  - mobi
+  - odt
+  - pdb
+  - pdf
+  - prc
+  - rtf
+  - txt
+ReleaseDate: 2022-07-16
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/kovidgoyal/calibre/releases/download/v6.2.1/calibre-64bit-6.2.1.msi
+    InstallerSha256: C95AB094CDF954C7D54C887434B5D77DF121ED2A09BF1840712E7A1B23EB308A
+    ProductCode: "{9DA78E49-78D0-4CEF-9B93-0E1EE35B6FB4}"
```

```suggestion
    ProductCode: "{D735F083-9DD5-40A6-B7E1-6C833E3F6FC3}"
```
winget-create / YamlCreate.ps1 will automatically do this for you. It can pull the GUID from the MSI Database.